Sources/Honeycomb/UIKit/UIViewController.swift

```diff
@@ -4,6 +4,27 @@
     import UIKit
 
     extension UIViewController {
+        var storyboardId: String? {
+            return value(forKey: "storyboardIdentifier") as? String
+        }
```

Comment:
AFAICT, Apple doesn't actually document this, but [StackOverflow to the rescue](https://stackoverflow.com/a/68632047) again

Reply:
In my experience with fields like this, it falls into a gray area. You're not really _supposed_ to do it, but you're not really supposed to swizzle either, and it may break at any time. I don't really have a problem with us doing it, as one of several possible names, but I wouldn't want to rely on it too highly either... I'll reference this in the other comments.